app/views/components/docs/figure.yml

```diff
@@ -0,0 +1,20 @@
+name: Figure
+description: A responsive image
+body: |
```

Comment:
We might want to include details about:

- alt text should be included
- when a figure should be used, maybe linking to https://developer.mozilla.org/en/docs/Web/HTML/Element/figure